projects/tomcat/build.sh

```diff
@@ -19,10 +19,12 @@ cp -r "/usr/lib/jvm/java-17-openjdk-amd64/" "$JAVA_HOME"
 
 $ANT
 
-cp "output/build/lib/tomcat-coyote.jar" "$OUT/tomcat-coyote.jar"
-cp "output/build/lib/tomcat-util.jar" "$OUT/tomcat-util.jar"
+cd $SRC/tomcat/output/classes && jar cfv classes.jar . && mv ./classes.jar $OUT && cd $SRC/tomcat
```

Comment:
Is there a reason why you added this? No big deals but you don't have to pack all you jars together, you can just add them one by one in the `ALL_JARS` variable

Reply:
I was just thinking that if I do this then I don't need to change this part in build.sh that much anymore, but maybe it slows down the run, I will change it back